plugins/coverimage.koplugin/main.lua

```diff
@@ -349,7 +348,7 @@ function CoverImage:addToMainMenu(menu_items)
                         end,
                     },
                     {
-                        text = _("Original image"),
+                        text = _("Original size"),
```

Comment:
Are you sure about this change ? The callback seems to be about background.

Reply:
Something like "original page" might be clearer?

Reply:
Given the wider context, I think the current wording is correct (in the sense that it's the image, and only the image).

The wording is not limited to saying "no background" or "keep bg as-is" (or however we word that in ScreenSaver) because, AFAICT, it *also* bypasses scaling, and just dumps the file as-is.

Reply:
TL;DR: Yeah, ixnay on that hunk ;).